.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies, if possible)_.
+
+### Affected node.js version(s)
+
+_Please fill in affected version(s)_
```

Comment:
As to the length/weight of the template: Something like:

`Version:  _enter affected node.js version(s)_` 

…would be more compact